Makefile

```diff
@@ -216,12 +216,7 @@ basic-test: install-tools
 ci-test-job: install-tools dashboard-ui
 	@$(FAILPOINT_ENABLE)
 	CGO_ENABLED=1 go test -timeout=15m -tags deadlock -race -covermode=atomic -coverprofile=covprofile -coverpkg=./... $(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX))
-	@$(FAILPOINT_DISABLE)
-
-ci-test-job-submod: install-tools dashboard-ui
-	@$(FAILPOINT_ENABLE)
-	@ for mod in $(SUBMODULES); do cd $$mod && $(MAKE) ci-test-job && cd $(ROOT_PATH) > /dev/null && cat $$mod/covprofile >> covprofile; done
-	@$(FAILPOINT_DISABLE)
+	@ for mod in $(shell ./scripts/ci-subtask.sh $(JOB_COUNT) $(JOB_INDEX) 1); do cd $$mod && $(MAKE) ci-test-job && cd $(ROOT_PATH) > /dev/null && cat $$mod/covprofile >> covprofile; done	
```

Comment:
Do we also need `@$(FAILPOINT_DISABLE)` or just by `clean`?

Reply:
This task is only used for the CI enviroment, so there is no need to do the clean-up work.